web_src/js/features/stopwatch.js

```diff
@@ -8,20 +9,25 @@ export function initStopwatch() {
     return;
   }
 
-  const stopwatchEl = $('.active-stopwatch-trigger');
+  const stopwatchEl = document.querySelector('.active-stopwatch-trigger');
+  const stopwatchPopup = document.querySelector('.active-stopwatch-popup');
 
-  if (!stopwatchEl.length) {
+  if (!stopwatchEl || !stopwatchPopup) {
     return;
   }
 
-  stopwatchEl.removeAttr('href'); // intended for noscript mode only
-  stopwatchEl.popup({
-    position: 'bottom right',
-    hoverable: true,
+  stopwatchEl.removeAttribute('href'); // intended for noscript mode only
+
+  createTippy(stopwatchEl, {
+    content: stopwatchPopup,
+    placement: 'bottom-end',
+    trigger: 'click',
+    maxWidth: 'none',
+    interactive: true,
   });
 
   // form handlers
-  $('form > button', stopwatchEl).on('click', function () {
+  $('form > button', $(stopwatchEl)).on('click', function () {
```

Comment:
Could this be `.active-stopwatch-trigger form > button` ?

Reply:
I removed this click handler and replaced it with `button[type=submit]`, which does the same without JS.